spec/cli_spec.rb

```diff
@@ -431,6 +447,48 @@
       end
     end
 
+    context "#local_database?" do
+      it "should return false when no database given" do
+        expect(subject).to receive(:database?).and_return(false)
```

Comment:
I would prefer all of these stubs be done by just setting the options like what you're doing on [line 464](https://github.com/ManageIQ/manageiq-appliance_console/pull/20/files#diff-9f2c191ddaf44048040f9babca4ec4caR464)

Reply:
Thanks, but I'm not agree with this, since we unit test `local_database?`, I think it's better to assume `database?` and other stubs works correctly. For whether `database?` work correctly or not is should be tested in their own context. Is this understanding make sense?

Reply:
Makes sense to me.